src/app/components/CustomButton/index.tsx

```diff
@@ -10,18 +10,29 @@ import styles from './styles';
 const CustomButton = (props: CustomButtonProps) => {
   const customStyles = useCallback(() => getCustomStyles(VARIANTS, props, styles), [props]);
   const customTextStyles = useCallback(() => getCustomStyles(VARIANTS, props, styles, 'Content'), [props]);
-  const { onPress, style, activeOpacity, icon, title, iconStyle, disabled, textStyle, testID, ...textProps } =
-    props;
+  const {
+    onPress,
+    style,
+    activeOpacity,
+    icon,
+    title,
+    iconStyle,
+    disabled,
+    textStyle,
+    testID,
+    primary,
```

Comment:
be careful because in this [PR](https://github.com/Wolox/conduit-react-native/pull/4/files) changes were also made in the same file